cmd/shell.go

```diff
@@ -316,3 +317,41 @@ func LoadCert(path string) (cert []byte, err error) {
 	cert = block.Bytes
 	return
 }
+
+var openConnections int64
+
+// HandlerTimer monitors HTTP performance and sends the details to StatsD.
+func HandlerTimer(handler http.Handler, stats statsd.Statter, prefix string) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
This was previously defined in three separate binary definition `cmd/...` files which don't get much attention, I thought it might be better to just define this once in a lib they all import anyway.